docs/api-guide.md

```diff
@@ -760,8 +836,8 @@ public class Sample {
         } catch (RollbackException ex) {
           // Rolling back the transaction failed. You can log it here
         }
-      } catch (CrudException | CommitException e) {
-        // If you catch CrudException or CommitException, it indicates some failure happens, so you
+      } catch (CrudException | CommitUnsatisfiedConditionException | CommitException e) {
```

Comment:
The handling of `CommitUnsatisfiedConditionException` is very dependent on the application use cases. Therefore, we should use a separate `catch` block for this exception, as follows:

```suggestion
      } catch (CommitUnsatisfiedConditionException e) {
        // If you catch CommitUnsatisfiedConditionException, ...
      } catch (CrudException | CommitException e) {
```

Reply:
Understood, I put the new `UnsatisfiedConditionException` in a separate catch block. Thank you.
Fixed in https://github.com/scalar-labs/scalardb/pull/899/commits/46441d23a7772be58cd2ad2bfb887b71767f5b31